src/AbstractConfigFactory.php

```diff
@@ -52,11 +43,10 @@ public function canCreateServiceWithName(ServiceLocatorInterface $serviceLocator
     /**
      * Determine if we can create a service (SM v3)
      *
-     * @param ContainerInterface $container
      * @param string $requestedName
      * @return bool
      */
-    public function canCreate(ContainerInterface $container, $requestedName)
+    public function canCreate(containerinterface $container, $requestedName)
```

Comment:
This case sensitivity change is a bug in PHPCS, I think

Reply:
yep

```
FILE: src/AbstractConfigFactory.php
----------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------------
 5 | ERROR | [x] Expected class name interop\container\containerinterface; found Interop\Container\Containerinterface
----------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------------------------------------------------------

```

Reply:
i ignore this in phpcs.xml